modules/lib/src/main/resources/assets/js/app/wizard/page/LiveFormPanel.ts

```diff
@@ -516,9 +516,17 @@ export class LiveFormPanel
         saveAction.onExecuted(() => {
             const selectedItem = this.lastInspectedItemPath ? PageState.getComponentByPath(this.lastInspectedItemPath) : null;
 
+            if (selectedItem instanceof DescriptorBasedComponent) {
+                this.contentWizardPanel.setMarkedAsReady(false);
```

Comment:
Why?

Reply:
To signal that we are not loading a new component but reloading existing, thus even though it has page contributions we won't reload entire page